test/integration/otel/metrics.test.js

```diff
@@ -8,18 +8,23 @@
 const test = require('node:test')
 const assert = require('node:assert')
 const https = require('node:https')
+const path = require('node:path')
 const { once } = require('node:events')
+const protobuf = require('protobufjs')
 
 const fakeCert = require('#testlib/fake-cert.js')
-const promiseResolvers = require('#testlib/promise-resolvers.js')
 const helper = require('#testlib/agent_helper.js')
 
 test.beforeEach(async (ctx) => {
   ctx.nr = {}
   ctx.nr.agent = helper.instrumentMockedAgent({
     opentelemetry_bridge: {
       enabled: true,
-      metrics: { enabled: true }
+      metrics: {
+        enabled: true,
+        exportInterval: 4_000,
```

Comment:
can't this be a lot smaller since we mock out the agent?

Reply:
we immediately emit `started` when we set up the mock agent [here](https://github.com/newrelic/node-newrelic/blob/77e7e0a27f719ce14d31636e46600ec7ff881b13/test/lib/agent_helper.js#L112)